db/migrate/20190221131741_delete_account_invoices_adjustments.rb

```diff
@@ -0,0 +1,11 @@
+class DeleteAccountInvoicesAdjustments < ActiveRecord::Migration
+  def up
+    Spree::Adjustment
+      .where(:source_type => 'BillablePeriod')
```

Comment:
don't you have Rubocop's feedback integrated into your editor? I strongly recommend it. We need to stop using hashrocket syntax :see_no_evil:

Reply:
I do, and I use it a lot, for some reason it's not showing up on migrations...
new commit to fix this.